packages/scaffold-config/src/ct-detect-third-party.ts

```diff
@@ -18,54 +19,41 @@ const DependencyArraySchema = z.array(DependencySchema)
 
 const BundlerSchema = z.enum(['webpack', 'vite'])
 
+const thirdPartyDefinitionPrefixes = {
+  // matches @org/cypress-ct-*
+  namespacedPrefixRe: /^@.*?\/cypress-ct.*/,
+  globalPrefix: 'cypress-ct-',
+}
+
+export function isThirdPartyDefinition (definition: Cypress.ComponentFrameworkDefinition | Cypress.ThirdPartyComponentFrameworkDefinition): definition is Cypress.ThirdPartyComponentFrameworkDefinition {
+  return definition.type.startsWith(thirdPartyDefinitionPrefixes.globalPrefix) ||
+    Boolean(definition.type.match(thirdPartyDefinitionPrefixes.namespacedPrefixRe))
+}
+
 const ThirdPartyComponentFrameworkSchema = z.object({
-  type: z.string().startsWith('cypress-ct-'),
+  type: z.string().startsWith(thirdPartyDefinitionPrefixes.globalPrefix).or(z.string().regex(thirdPartyDefinitionPrefixes.namespacedPrefixRe)),
   name: z.string(),
   supportedBundlers: z.array(BundlerSchema),
   detectors: DependencyArraySchema,
   dependencies: z.function(),
   componentIndexHtml: z.optional(z.function()),
 })
 
-const CT_FRAMEWORK_GLOB = path.join('node_modules', 'cypress-ct-*', 'package.json')
-
-// tsc will compile `import(...)` calls to require unless a different tsconfig.module value
-// is used (e.g. module=node16). To change this, we would also have to change the ts-node behavior when requiring the
-// Cypress config file. This hack for keeping dynamic imports from being converted works across all
-// of our supported node versions
-
-// const _dynamicImport = new Function('specifier', 'return import(specifier)')
-
-// const dynamicImport = <T>(module: string) => {
-//   return _dynamicImport(module) as Promise<T>
-// }
-
-// const dynamicAbsoluteImport = (filePath: string) => {
-//   return dynamicImport(pathToFileURL(filePath).href) as Promise<any>
-// }
-
-/**
- * When compiling CJS -> ESM, TS can produce:
- * Imported [Module: null prototype] { __esModule: true, default: { default: { type: 'cypress-ct-solid-js', ... } } }
- * We just keep getting `default` property until none exists.
- */
-function getDefaultExport<T extends { default?: T }> (mod: T): T {
```

Comment:
We do not need this since we stopped using `dynamicAbsoluteImport` - that's already commented out.